docs/man/rpmkeys.8.scd

```diff
@@ -6,76 +6,128 @@ rpmkeys - RPM Keyring
 
-The general forms of *rpm*(8) digital signature commands are
+*rpmkeys* is used for manipulating the *rpm* keyring and verifying
+package digital signatures with the contained keys.
 
-*rpmkeys* {*-l*|*--list*} [_FINGERPRINT_ ...]
+For all available operations, see *OPERATIONS*.
 
-*rpmkeys* {*-x*|*--export*} [_FINGERPRINT_ ...]
+# OPERATIONS
 
-*rpmkeys* {*-i*|*--import*} _PUBKEY_ ...
+*-K*,
+*--checksig*
+	Verify the digests and signatures contained in _PACKAGE_FILE_ to
+	ensure the integrity and origin of the package.
 
-*rpmkeys* {*-e*|*--erase*|*-d*|*--delete*} _FINGERPRINT_ ...
+*-d*,
+*-e*,
+*--delete*,
+*--erase*
+	Erase the key(s) designated by _FINGERPRINT_.
 
-*rpmkeys* {*-K*|*--checksig*} _PACKAGE_FILE_ ...
+*-x*,
+*--export*
+	Output the key(s) designated by _FINGERPRINT_ using an ASCII-armor
+	encoding.  If _FINGERPRINT_ is not specified, output all keys.
 
-*rpmkeys* {*--rebuild*} [*--from* _BACKEND_]
+*--import*
+	Import ASCII-armored public keys.
+	Digital signatures cannot be verified without the corresponding
+	public key (aka certificate).
 
-The *--checksig* option checks all the digests and signatures
-contained in _PACKAGE_FILE_ to ensure the integrity and origin of the
-package. Note that signatures are now verified whenever a package is
-read, and *--checksig* is useful to verify all of the digests and
-signatures associated with a package.
+*-l*,
+*--list*
+	List currently imported public key(s) (aka certificates) by their
+	fingerprint and user ID. If no fingerprints are specified, list all
+	keys.
 
-Digital signatures cannot be verified without a public key. An ASCII
-armored public key can be added to the *rpm* persistent keyring using
-*--import*.
+*--rebuild*
+	Recreate the public key storage. Update to the latest format and drop
+	unreadable keys.
 
-The following commands are available for manipulating the persistent
-rpm keyring:
+# ARGUMENTS
 
-*rpmkeys* {*-l*|*--list*} [_FINGERPRINT_...]
+_FINGERPRINT_
+	The handle used for all operations on the keys.
+_PACKAGE_FILE_
+	An *rpm* package file or a manifest.
+_PUBKEY_
+	An ASCII-armored OpenPGP public key (aka certificate).
 
-List currently imported public key(s) (aka certificates) by their
-fingerprint and user ID. If no fingerprints are specified, list all keys.
+# OPTIONS
 
-The fingerprint is the handle used for all operations on the keys.
+See *rpm.common*(8) for the options common to all *rpm* executables.
 
-*rpmkeys* {*-x*|*--export*} [_FINGERPRINT_ ...]
+# REBUILD OPTIONS
 
-Output the key(s) using an ASCII-armor encoding.
+*--from* <*fs*|*openpgp*|*rpmdb*>
 
-Exporting allows for inspecting the data with specialized tools, such
-as Sequoia or GnuPG. For example:
+Use the keys from the specified backend to rebuild the currently
+configured keystore backend.
+This can be used to convert from one key storage to another.
 
-*rpmkeys --export 771b18d3d7baa28734333c424344591e1964c5fc | sq inspect *
+# OUTPUT
+*--checksig*
+	Non-verbose output:
```

Comment:
How about dropping this line and just listing the non-verbose output "by default", with the verbose variant below?

Reply:
The idea being, it would make the style more consistent with the operations that only have one type of output.